test/test_prototype_transforms.py

```diff
@@ -2377,20 +2386,23 @@ def test_sanitize_bounding_boxes(min_size, labels_getter):
     valid_indices = [i for (i, is_valid) in enumerate(is_valid_mask) if is_valid]
 
     boxes = torch.tensor(boxes)
-    labels = torch.arange(boxes.shape[-2])
+    labels = torch.arange(boxes.shape[0])
 
     boxes = datapoints.BoundingBox(
         boxes,
         format=datapoints.BoundingBoxFormat.XYXY,
         spatial_size=(H, W),
     )
 
+    masks = datapoints.Mask(torch.randint(0, 2, size=(boxes.shape[0], H, W)))
```

Comment:
Note that adding a batch dim like below fails. Coming back to this @pmeier , should we allow for the batch dim to be here (and enforce batch_size == 1) ?

```suggestion
    masks = datapoints.Mask(torch.randint(0, 2, size=(1, boxes.shape[0], H, W)))
```

Reply:
As discussed offline, there is no use case for that yet, since on the sample level, one cannot create batch bounding box tensors. For example, let's say you have two samples, but the first contains and image with 5 objects and the second one has 8 objects. So the shapes of the bounding box tensors will be `(5, 4)` and `(8, 4)` and thus cannot be batched into a single tensors. Thus, in the detection context a batch of samples is just a list.